.github/workflows/linters.yml

```diff
@@ -0,0 +1,45 @@
+name: Linters
```

Comment:
<img width="850" alt="Screenshot 2022-11-28 at 21 41 41" src="https://user-images.githubusercontent.com/69011963/204332945-b475ea99-df35-4ca9-ba53-ee446f1090f8.png">

Reply:
HI @Rustamxon7, the errors are already solved... Regards.